dolphinscheduler-ui/src/router/modules/ui-setting.ts

```diff
@@ -30,11 +30,11 @@ export default {
   children: [
     {
       path: '',
-      name: 'ui-setting',
+      name: 'ui-setting2',
```

Comment:
Please use another PR to do this change.

Reply:
Removed modifications